docs/src/pages/getting-started/faq/faq.md

```diff
@@ -126,7 +126,25 @@ export default withTheme(withStyles(styles)(Modal));
 
 ## How can I access the DOM element?
 
-Wrap the component with the [`RootRef`](/api/root-ref/) helper.
+All Material-UI components that should render something in the DOM forward their
+ref to the underlying built-in component. This means that you can get DOM elements
```

Comment:
DOM elements don't know anything about refs though. I think the current text is technically correct because we end up with e.g. `<div ref={ref} />` which mens the ref is attached to the built-in component not the the DOM element.

maybe stick with `host component` here. I've rarely seen "built-in component"

 ```suggestion
ref to the underlying host component. This means that you can get DOM elements
```

The important thing is that the takeway should be: "How do I get the DOM element? Just attach a ref. Material-UI will take care of the rest"